webapp/lib/controller.dart

```diff
@@ -6,13 +6,15 @@ import 'logger.dart';
 import 'platform.dart' as platform;
 import 'view.dart' as view;
 import 'router.dart';
+import 'model.dart' as model;
 
 Logger log = new Logger('controller.dart');
 Router router;
 
 enum UIAction {
   userSignedIn,
-  loadProjectConfiguration,
+  loadBatchRepliesConfiguration,
+  loadEscalatesConfiguration,
```

Comment:
consider referring to these as packages, e.g.
```suggestion
  loadBatchRepliesPackageConfiguration,
  loadEscalatesPackageConfiguration,
```

Reply:
Renamed.